specs/deneb/fork-choice.md

```diff
@@ -82,7 +82,11 @@ def on_block(store: Store, signed_block: SignedBeaconBlock) -> None:
     finalized_slot = compute_start_slot_at_epoch(store.finalized_checkpoint.epoch)
     assert block.slot > finalized_slot
     # Check block is a descendant of the finalized block at the checkpoint finalized slot
-    assert get_ancestor(store, block.parent_root, finalized_slot) == store.finalized_checkpoint.root
+    assert store.finalized_checkpoint.root == get_ancestor_at_epoch_boundary(
+        store,
+        block.parent_root,
+        store.finalized_checkpoint.epoch,
+    )
```

Comment:
Doing an assignment before the assert is preferable to reduce line count and increase readability. ditto at a few other places in this patch

```suggestion
    epoch_boundary_root = get_ancestor_at_epoch_boundary(store, block.parent_root, store.finalized_checkpoint.epoch)
    assert store.finalized_checkpoint.root == epoch_boundary_root
```

Reply:
Done here in all other places marked with "ditto"